spec/controllers/spree/admin/orders/invoices_spec.rb

```diff
@@ -103,4 +103,43 @@
       end
     end
   end
+
+  describe "#index" do
+    let!(:user) { create(:user) }
+    let!(:enterprise_user) { create(:user) }
```

Comment:
There's no difference between the two users. There are two identical test cases for each user. I think that you need to give the enterprise user an enterprise.